lib/sinatra/shopify-sinatra-app.rb

```diff
@@ -189,7 +190,9 @@ def self.registered(app)
         provider :shopify,
                  app.settings.api_key,
                  app.settings.shared_secret,
-
+                 #Fixes for new Shopify callback issues
+                 redirect_uri: app.settings.hostname + "/auth/shopify/callback",
+                 callback_url: app.settings.hostname + "/auth/shopify/callback",
```

Comment:
Good point - can you add it here? https://github.com/kevinhughes27/shopify-sinatra-app/blob/master/bin/shopify-sinatra-app-generator#L42

I am not really a fan of the generator in this repo and I might remove it soon but we might as well add it for now